simulation:
  freq: 400.0
  stand_kp: 100.0
  stand_kd: 1.0
  joint_kp: 3.0
  joint_kd: 0.2
robot:
  freq: 400.0
  stand_kp: 100.0
  stand_kd: 1.0
  joint_kp: 3.0
  joint_kd: 0.2
